src/ServiceControl/Infrastructure/Installers/AuditQueueInstaller.cs

```diff
@@ -1,18 +0,0 @@
-namespace ServiceBus.Management.Infrastructure.Installers
-{
-    using NServiceBus;
-    using NServiceBus.Unicast.Queuing;
-    using Settings;
-
-    public class AuditQueueInstaller : IWantQueueCreated
```

Comment:
I'm pretty sure this is a leftover which is no longer needed:

- Error queue doesn't have an equivalent counterpart, still, it gets created
- Even when I remove this file the audit queue gets created